<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="WKWIN6984194" failures="1" tests="5" name="Surefire test" time="9.977" errors="0" timestamp="2021-01-25T22:29:41 IST">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="2.887"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="1.219"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.716"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.484">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at stepdefinitions.GetWeatherDetails.verify_count_is_equal_to_for_the_given_place(GetWeatherDetails.java:85)
at âœ½.Verify "WEOID" count is Equal to 1 for the given place(file:///C:/BDD_Workspace/MetaWeatherAPITestProject/src/test/resources/AppFeatures/VerifyPlaceID.feature:7)
... Removed 4 stack frames]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.23"/>
</testsuite> <!-- Surefire test -->
